app/controllers/api/v1/rents_controller.rb

```diff
@@ -2,11 +2,15 @@ module Api
   module V1
     class RentsController < ApplicationController
       def index
-        render_paginated @current_user.rents
+        rents = policy_scope(Rent)
+        authorize rents
```

Comment:
I think this could be removed, you are already filtering the retns in the policy scope

Reply:
Done